tools/src/zopensetupmain.c

```diff
@@ -35,7 +36,7 @@ static void syntax(const char* pgm, const char** bootpkg) {
     pgm = &base[1];
   }
   fprintf(stderr, "%s : Install the z/OS Open Source Tools 'starter' environment by downloading from %s\n"
-                  "Syntax: %s [-vq] <root>\n"
+                  "Syntax: %s [-vq] [<root>]\n"
```

Comment:
```suggestion
                  "Syntax: %s [-vq] [root directory]\n"
```
<> usually means required argument, so I would suggestion removing it